.github/workflows/test-action.yaml

```diff
@@ -2,11 +2,34 @@ name: Test Action
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+      - main
   pull_request:
     branches:
       - main
 
 jobs:
+
+  automated-tests:
+    runs-on: ubuntu-latest
```

Comment:
Good practice to set a `timeout-minutes` key to avoid waiting 6 hours (default) if something goes wrong:

```
timeout-minutes: 10 or 15
```

Reply:
you're right! done in https://github.com/heal-dev/trigger/pull/7.